src/server/util/WebhookManager.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * @author kecso / https://github.com/kecso
+ */
+var MessageSender = require('webhook-manager/src/hookMessager'),
+    CONSTANTS = requireJS('common/storage/constants'),
+    spawn = require('child_process').spawn,
+    path = require('path'),
+    __baseDir = requireJS.s.contexts._.config.baseUrl; // TODO: this is ugly
```

Comment:
`require.resolve('webhook-manager')` will give you its path in a safe way when calling it as a subprocess.